lib/Matrix.pm

```diff
@@ -68,6 +74,17 @@ sub _stringify {
     return($s);
 }
 
+=head3 Accessor functions
+	
+	(these are deprecated for direct use.  Use the covering Methods
+	 provided by MathObject Matrices instead.)
+	 
+	L($matrix) - return matrix L of the LR decomposition
+	R($matrix) - return matrix R of the LR decomposition
+	PL($matrix) return 
+	PR($matrix 
+
```

Comment:
Is there something missing in the above two lines?

Reply:
Good catch.  I'd also missed a =cut which would have wiped out the definition of the L and R 
decomposition functions. Here is the whole segment in context:
-----------------
=head3 Accessor functions
	
	(these are deprecated for direct use.  Use the covering Methods
	 provided by MathObject Matrices instead.)
	 
	L($matrix) - return matrix L of the LR decomposition
	R($matrix) - return matrix R of the LR decomposition
	PL($matrix) - return permutation matrix
	PR($matrix) - return permutation matrix
	Original matrix is  PL * L * R *PR = M 
	
Obtain the Left Right matrices of the decomposition 
and the two pivot permutation matrices
the original is M = PL*L*R*PR

=cut